packages/now-cli/src/util/env/add-env-record.ts

```diff
@@ -1,60 +1,34 @@
 import { Output } from '../output';
 import Client from '../client';
 import { Secret, ProjectEnvTarget, ProjectEnvVariableV5 } from '../../types';
-import { customAlphabet } from 'nanoid';
-import slugify from '@sindresorhus/slugify';
 
 export default async function addEnvRecord(
   output: Output,
   client: Client,
   projectId: string,
-  envName: string,
-  envValue: string | undefined,
+  type: 'plain' | 'secret' | 'system',
```

Comment:
```suggestion
  type: ProjectEnvType,
```

Should this use the Enum isntead?

Reply:
Fixed in https://github.com/vercel/vercel/pull/5372/commits/052ae16abbb2cc101bc0d7e113d271bffd012519 👍